packages/ui/cart/drawer/CartDrawer.tsx

```diff
@@ -152,14 +169,7 @@ export function CartDrawer() {
                            <CrossSell />
                         </>
                      )}
-                     <Fade
-                        show={isCartEmpty}
-                        disableExitAnimation
-                        position="absolute"
-                        w="full"
-                        top="0"
-                        left="0"
-                     >
+                     <Fade show={isCartEmpty} disableExitAnimation w="full">
```

Comment:
@dhmacs is this ok to change? https://github.com/iFixit/react-commerce/commit/7d2a8bbf70bf878da4844912672fcf9bce1c936e

I couldn't tell a difference animation wise personally, but i also don't know what i was looking for exactly

Reply:
I think the purpose was that the cart empty state would animate in place rather than slide up from below, but I've tested it and you're right, doesn't seem to make any difference really. I think it's ok to drop it 👍